packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -56,12 +56,12 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
     let { labelText, className, disabled, underlined, getStyles, theme } = this.props;
     let { value, hasFocus, id } = this.state;
 
-    const classNames = getClassNames(getStyles!, { theme: theme!, underlined, hasFocus, disabled });
+    const classNames = getClassNames(getStyles!, { theme: theme!, className, underlined, hasFocus, disabled });
 
     return (
       <div
         ref={ this._resolveRef('_rootElement') }
-        className={ css(className, classNames.root,
+        className={ css(classNames.root,
           hasFocus && 'is-active',
           disabled && 'is-disabled',
```

Comment:
I believe the activee, disabled, etc ones get applied here either.  For example,  we don't want then in the DOM for Office Online,  get styles lets us override this behavior

Reply:
In general, I don't think merge style components use the css helper, but  check the other examples